mkosi/__init__.py

```diff
@@ -115,9 +116,9 @@
 SomeIO = Union[BinaryIO, TextIO]
 PathString = Union[Path, str]
 
+MKOSI_COMMANDS_NEED_BUILD = (Verb.shell, Verb.boot, Verb.qemu, Verb.serve)
+MKOSI_COMMANDS_SUDO = (Verb.build, Verb.clean, Verb.shell, Verb.boot, Verb.qemu, Verb.serve)
 MKOSI_COMMANDS_CMDLINE = ("build", "shell", "boot", "qemu", "ssh")
-MKOSI_COMMANDS_NEED_BUILD = ("shell", "boot", "qemu", "serve")
-MKOSI_COMMANDS_SUDO = ("build", "clean", "shell", "boot", "qemu", "serve")
 MKOSI_COMMANDS = ("build", "clean", "help", "summary", "genkey", "bump", "serve") + MKOSI_COMMANDS_CMDLINE
```

Comment:
`MKOSI_COMMANDS` is only used once in parse_args(), you can change that usage to `for verb in Verb:` and get rid of `MKOSI_COMMANDS`.

Reply:
Good catch